client/overview/index.js

```diff
@@ -9,12 +9,16 @@
 import Page from 'components/page';
 import { TestModeNotice, topics } from 'components/test-mode-notice';
 import TaskList from './task-list';
+import { getTasks } from './task-list/tasks';
+import './style.scss';
 
 const OverviewPage = () => {
+	const { accountStatus, showUpdateDetailsTask } = wcpaySettings;
+	const tasks = getTasks( { accountStatus, showUpdateDetailsTask } );
 	return (
-		<Page>
+		<Page className="overview">
 			<TestModeNotice topic={ topics.overview } />
-			<TaskList />
+			{ tasks.length && <TaskList tasks={ tasks } /> }
```

Comment:
This might have to turn into a ternary statement, as it renders a `0` for me now 😆 
<img width="387" alt="Screen Shot 2021-05-17 at 9 47 28 AM" src="https://user-images.githubusercontent.com/2240960/118491015-f5614480-b6f4-11eb-8570-2685617c73ab.png">

Reply:
Nice catch. Fixed in e79e042.